### PR TITLE
Create zh_CN.lang

### PR DESCRIPTION
**Pull request by Koud-Wind**

### CODE REVIEW

**Comment by Koud-Wind** [comment]
In addition, turning off the world and screen shaders by default will result in more frames, allowing for the use of light and shadow at any time without the need to manually turn them off

**Comment by SticksDeveloper** [comment]
Because I cannot check for sure, You are sure all of these are all translated not using google translate or any translator of sorts and come from the actual language?

**Comment by Koud-Wind** [comment]
I'm sure, If there are translation errors, I will correct them again

**Comment by Koud-Wind** [comment]
But there are some prompts on my server that I can only rely on the machine to translate Chinese into English

File: src/main/java/com/paneedah/weaponlib/config/ModernConfigManager.java [hunk 1]
```diff
@@ -48,10 +48,10 @@ public class ModernConfigManager {
 	public static boolean enableAllShaders = true;
 
 	@ConfigSync(category = CATEGORY_RENDERING, comment = "Setting this to false disables all world shaders, enabling allows to select which world shaders are used.")
-	public static boolean enableWorldShaders = true;
+	public static boolean enableWorldShaders = false;
 	
 	@ConfigSync(category = CATEGORY_RENDERING, comment = "Setting this to false disables all screen shaders, enabling allows to select which screen shaders are used.")
-	public static boolean enableScreenShaders = true;
+	public static boolean enableScreenShaders = false;
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/weaponlib/config/ModernConfigManager.java:54]: Please don't do that

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/weaponlib/config/ModernConfigManager.java (hunk 1 above)
Comment (Koud-Wind) [reply src/main/java/com/paneedah/weaponlib/config/ModernConfigManager.java]: > Please don't do that

I fixed it

**Comment by Desoroxxx** [comment]
Is there any modification you want to do or can this be merged?

**Comment by Koud-Wind** [comment]
There is nothing for the time being

**Comment by Koud-Wind** [comment]
In addition, I have a source code about the VMW0.6 explosion effect, which will not damage the terrain (including flash grenades). I hope you can modify it

[Explosion.zip](https://github.com/Cubed-Development/Modern-Warfare-Cubed/files/11454820/Explosion.zip)

**Pull request merged**